src/session/session_api.c

```diff
@@ -1668,14 +1669,25 @@ __wt_session_range_truncate(
     trunc_info->orig_stop_key = orig_stop_key;
     trunc_info->uri = actual_uri;
 
+    if (start != NULL && strcmp(start->uri, WT_HS_URI) == 0)
```

Comment:
Can we use WT_IS_HS macro?

Reply:
With the new approach we no longer need changes in `__wt_session_range_truncate`.
@pmacko86 I like your suggestion of `WT_CURSOR_IS_HS` macros. I'll add it to the scope of WT-13189